website/docs/r/identity_group.html.md

```diff
@@ -60,6 +60,8 @@ The following arguments are supported:
 
 * `external_policies` - (Optional) `false` by default. If set to `true`, this resource will ignore any policies return from Vault or specified in the resource. You can use [`vault_identity_group_policies`](identity_group_policies.html) to manage policies for this group in a decoupled manner.
```

Comment:
I know it's not your change, but it would be kind if you could change `will ignore any policies return` with `will ignore any policies returned` here. Thanks!

Reply:
Fixed

Reply:
Thank you!